telosnetwork/open-block-explorer
# 452 autologinlogout not handled correctly

# Fixes #452 #398 

## Description
This PR fixes and streamlines the login/logout logic and balance loading to be more seamless while also replacing unnec hyperion references for account data with nodeos. 

## Test scenarios

1. Log in via Anchor
2. click 'Wallet' in subheader
3. see balances and send/resource/staking buttons
4. click each to make sure modals load
5. click 'disconnect' (logout) 
6. see page _not_ refresh, buttons and 'Wallet' subheader option hidden
7. while on account page, log back into same account
8. see buttons and wallet option re-appear 
9. verify total, liquid, price @, rex and staked balances, resource stats
10. test new proposal creation (should still work as before)
11. test browser navigation between logged in account and other accounts

## Checklist:
